src/test/java/org/jenkinsci/plugins/badge/extensions/SpecialValueParameterResolverExtensionTest.java

```diff
@@ -73,4 +75,37 @@ void testMultipleParameters() {
                 extension.resolve(mockRun, "buildId buildNumber duration startTime displayName description");
         assertThat(actualParameter, is("1234 1234 23:35 23:35 display name run description"));
     }
+
+    @Test
+    void resolveEmptyStringParameter() {
+        String actualParameter = extension.resolve(mockRun, "");
+        assertThat(actualParameter, is(""));
+    }
+
+    @Test
+    void resolveWithNullParameter() {
+        String actualParameter = extension.resolve(mockRun, null);
+        assertThat(actualParameter, is((String) null));
```

Comment:
Hamcrest has a method that does not require casting null to the desired type.
```suggestion
        assertThat(actualParameter, is(nullValue()));
```